reframe/core/schedulers/torque.py

```diff
@@ -0,0 +1,117 @@
+import re
+
+from reframe.core.config import settings
+from reframe.core.exceptions import JobError
+from reframe.core.logging import getlogger
+from reframe.core.schedulers.pbs import PbsJobScheduler, _run_strict
+from reframe.core.schedulers.registry import register_scheduler
+import reframe.utility.os_ext as os_ext
+
+
+JOB_STATES = {
+    'Q': 'QUEUED',
+    'H': 'HELD',
+    'R': 'RUNNING',
+    'E': 'EXITING',
+    'T': 'MOVED',
+    'W': 'WAITING',
+    'S': 'SUSPENDED',
+    'C': 'COMPLETED',
+}
+
+
+@register_scheduler('torque')
+class TorqueJobScheduler(PbsJobScheduler):
+    def _update_state(self, job):
+        '''Check the status of the job.'''
+
+        completed = os_ext.run_command('qstat -f %s' % job.jobid, log=False)
```

Comment:
Why do you disable logging of the command here? It can be useful for debugging.

Reply:
no particular reason, I'll enable it again if you prefer.